diff -Naur scdp-1.0b/acinclude.m4 scdp-1.0b-p/acinclude.m4
--- scdp-1.0b/acinclude.m4	2004-06-06 12:43:02.296449640 +0200
+++ scdp-1.0b-p/acinclude.m4	2004-06-06 12:43:22.977305672 +0200
@@ -6,7 +6,7 @@
 AC_DEFUN(AC_LIBNET_BASE,
 [ AC_MSG_CHECKING(Looking for libnet)
   define([AC_LIBNET_NAME], libnet_config)dnl
-  AC_PATH_PROG(AC_LIBNET_NAME, libnet-config)
+  AC_PATH_PROG(AC_LIBNET_NAME, libnet1-config)
   changequote(<<, >>)dnl
   if test -n "$ac_cv_path_<<>>AC_LIBNET_NAME"; then
     LIBNET_BASE=`echo $ac_cv_path_<<>>AC_LIBNET_NAME|sed 's%/[^/][^/]*//*[^/][^/]*$%%'`
diff -Naur scdp-1.0b/configure.in scdp-1.0b-p/configure.in
--- scdp-1.0b/configure.in	2001-03-20 21:16:48.000000000 +0100
+++ scdp-1.0b-p/configure.in	2004-06-06 12:45:36.827957280 +0200
@@ -16,8 +16,8 @@
 dnl Checks for libraries.
 dnl Find libnet configuration
 AC_LIBNET_BASE
-CFLAGS="-I${LIBNET_BASE}/include "`libnet-config --defines`' '$CFLAGS
-LIBS="-L${LIBNET_BASE}/lib "`libnet-config --libs`' '$LIBS
+CFLAGS="-I${LIBNET_BASE}/include "`libnet1-config --defines`' '$CFLAGS
+LIBS="-L${LIBNET_BASE}/lib "`libnet1-config --libs`' '$LIBS
 
 dnl Checks for header files.
 AC_HEADER_STDC
diff -Naur scdp-1.0b/scdp.c scdp-1.0b-p/scdp.c
--- scdp-1.0b/scdp.c	2001-03-20 21:16:47.000000000 +0100
+++ scdp-1.0b-p/scdp.c	2004-06-06 12:48:47.746933184 +0200
@@ -20,7 +20,7 @@
 
 #include <stdio.h>
 #include <conf.h>
-#include <libnet.h>
+#include <libnet1.h>
 #include <malloc.h>
 #include <sys/types.h>
 #include <sys/wait.h>
